migrations/20201012224257_add_max_attendees.ts

```diff
@@ -0,0 +1,16 @@
+import * as Knex from "knex";
```

Comment:
My eslint has been angry with the code:
![Screenshot 2020-10-13 01 05 10](https://user-images.githubusercontent.com/60111772/95798057-42111700-0cf1-11eb-967e-f2f980467721.png)